src/azul/security/auth.py

```diff
@@ -0,0 +1,80 @@
+# See https://allspark.dev.data.humancellatlas.org/dcp-ops/docs/wikis/Security/Authentication%20and%20Authorization/Setting%20up%20DCP%20Auth
+from logging import getLogger
+from typing import Dict, Any
+from urllib.parse import urlencode
+
+from azul import config
+from azul.security.verifier import (verify,
+                                    InvalidTokenError,
+                                    NonDecodableTokenError)
+
+logger = getLogger(__name__)
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```

Like everywhere else.